### PR TITLE
[ENG-2450] Fix email formatting problems

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose

Fix sporadic sentry errors from badly formatted emails.

## Changes

- pass user to moderator digest emails
- add colon to email template conditional

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify emails are sent correctly via registries moderation

## Documentation

🐞 fix, no docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-2450

### CODE REVIEW

**Comment by jwalz** [comment]
A few more missing context things that aren't addressed here:
None of the sanctions pass `reviewable` in the context, but they all reference `reviewable.provider` in the moderated case.

None of the sanctions pass `is_moderated` in the context in the non-authorizer case, but we still check it in the `non_admin` emails.

Embargo doesn't pass `is_moderated` in the authorizer case, either.

File: website/notifications/tasks.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ def _send_reviews_moderator_emails(send_type):
 
             mails.send_mail(
                 to_addr=user.username,
+                user=user,
```
Comment (brianjgeiger) [inline website/notifications/tasks.py:77]: Drive-by: Was it just the user's `fullname` that was needed here? Because if so, that's available under `name` on L80.

**Review by brianjgeiger — APPROVED** [review]
